akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -287,8 +286,8 @@ object HttpEntity {
     def dataBytes: Source[ByteString, Any] = data
 
     /**
-      * See [[HttpEntity#withSizeLimit]].
```

Comment:
Note: IntelliJ Scala plugin changed comment formatting to this style, and I don't know how to change it back. I correct these by hand but @sirthias might have missed it. Can the scalariform plugin fix these, btw?